spec/lib/stylesheet/manager_spec.rb

```diff
@@ -374,6 +374,16 @@ def manager(theme_id = nil)
         type_id: ThemeField.types[:theme_upload_var]
       )
 
+      upload2 = UploadCreator.new(image2, "icon.png").create_for(-1)
+      field = ThemeField.create!(
+        theme_id: theme.id,
+        target_id: Theme.targets[:common],
+        name: "icon",
+        value: "",
+        upload_id: upload2.id,
+        type_id: ThemeField.types[:theme_upload_var]
+      )
```

Comment:
I'm not sure I follow the spec... is `image2` a nil object?

Reply:
No, `image2` is not a nil object. The error is happening when the string and nil combination are found on the array. In this spec test, the upload record will create the nil value and the upload2 record will create the valid sha1 string.